app/javascript/packages/document-capture/components/in-person-prepare-step.tsx

```diff
@@ -9,33 +9,23 @@ import AnalyticsContext from '../context/analytics';
 import BackButton from './back-button';
 import InPersonTroubleshootingOptions from './in-person-troubleshooting-options';
 import { InPersonContext } from '../context';
+import InPersonPostOfficeOutageAlert from './in-person-post-office-outage-alert';
 
 function InPersonPrepareStep({ toPreviousStep }) {
   const { t } = useI18n();
   const { flowPath } = useContext(UploadContext);
   const { setSubmitEventMetadata } = useContext(AnalyticsContext);
   const { securityAndPrivacyHowItWorksURL } = useContext(MarketingSiteContext);
-  const { inPersonURL, inPersonCtaVariantActive } = useContext(InPersonContext);
+  const { inPersonURL, inPersonCtaVariantActive, inPersonUspsOutageMessageEnabled } =
+    useContext(InPersonContext);
 
   useEffect(() => {
     setSubmitEventMetadata({ in_person_cta_variant: inPersonCtaVariantActive });
   }, []);
 
   return (
     <>
-      <Alert type="warning" className="margin-bottom-4">
-        <>
-          <strong>
-            {t('idv.failure.exceptions.post_office_outage_error_message.post_cta.title')}
-          </strong>
-          <br />
-          <br />
-          {t('idv.failure.exceptions.post_office_outage_error_message.post_cta.body', {
-            app_name: getConfigValue('appName'),
-          })}
-          <br />
-        </>
-      </Alert>
+      {inPersonUspsOutageMessageEnabled && <InPersonPostOfficeOutageAlert />}
```

Comment:
I can understand going either way; the way you suggested is more compatible with some Redux/HOC component wrapping that I've seen. Since it does localize the enablement of the alert, I'll consider this in a follow-up PR.